deployability/modules/workflow_engine/examples/central_components/aws/dtt1-central_components-poc-aws.yaml

```diff
@@ -0,0 +1,69 @@
+version: 0.1
+description: This workflow is used to test manager deployment for DDT1 PoC
```

Comment:
the Wazuh manager

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5289/commits/7c1d2270fe036e068551866cd700baa9e1b5a2ea

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5289/commits/7c1d2270fe036e068551866cd700baa9e1b5a2ea